keras/src/ops/numpy_test.py

```diff
@@ -3966,6 +3966,13 @@ def test_round(self):
         self.assertAllClose(knp.round(x, decimals=-1), np.round(x, decimals=-1))
         self.assertAllClose(knp.Round(decimals=-1)(x), np.round(x, decimals=-1))
 
+    def test_searchsorted(self):
```

Comment:
Please also add a test for static shape inference (on KerasTensors).

Reply:
Done